src/scene/Light.cpp

```diff
@@ -297,7 +297,9 @@ EERIE_LIGHT * lightHandleGet(LightHandle lightHandle) {
 void lightHandleDestroy(LightHandle & handle) {
 	
 	if(EERIE_LIGHT * light = lightHandleGet(handle)) {
-		light->m_exists = false;
+		if(light) {
```

Comment:
This is redundant, the first if already checks the same.

Reply:
Probably a brain fart. Removed.